### PR TITLE
feat: add color and sync with bevy

### PR DESCRIPTION
**Pull request by lecoqjacob**
- Add color type to bones
- Add color type to Bones Sprite
- Sync with Bevy

### CODE REVIEW

**Review by zicklag — CHANGES_REQUESTED** [review]
Sweet! Could you switch these couple things to use the new `Color` type:

https://github.com/fishfolk/bones/blob/77e7960aff030a9fd235c273a174de8c16c40401/crates/bones_render/src/camera.rs#L54

https://github.com/fishfolk/bones/blob/77e7960aff030a9fd235c273a174de8c16c40401/crates/bones_render/src/line.rs#L10

**Comment by lecoqjacob** [comment]
> Sweet! Could you switch these couple things to use the new `Color` type:
> 
> 
> 
> https://github.com/fishfolk/bones/blob/77e7960aff030a9fd235c273a174de8c16c40401/crates/bones_render/src/camera.rs#L54
> 
> 
> 
> https://github.com/fishfolk/bones/blob/77e7960aff030a9fd235c273a174de8c16c40401/crates/bones_render/src/line.rs#L10

I'm on it!

File: crates/bones_render/src/camera.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //! Camera components.
 
-use crate::prelude::*;
+use crate::{color::Color, prelude::*};
```
Comment (zicklag) [inline crates/bones_render/src/camera.rs:3]: Could you add `Color` to the prelude.

And here, too:

https://github.com/fishfolk/bones/pull/110/files#diff-a755dd4667b7d1e7c74a892f3396c353b2945aef60faa131c460c197b764564eR3-R4

**Review by zicklag — CHANGES_REQUESTED** [review]
One more thing!

File: crates/bones_render/src/camera.rs (hunk 1 above)
Comment (lecoqjacob) [reply crates/bones_render/src/camera.rs]: silly me. that should have been there! nice catch

**Review by zicklag — APPROVED** [review]

**Pull request merged**